wolox-books/src/app/screens/auth/screens/components/book-card/book-card.component.html

```diff
@@ -0,0 +1,6 @@
+<img class="img" *ngIf="item.image_url; else defaultImg" [src]="item.image_url" [alt]="item.image_url">
```

Comment:
the alt should not be a url, it should be a name that represents the content of the image